Makefile

```diff
@@ -494,18 +494,15 @@ test-verif-proxy-wrapper:
 	CGO_CFLAGS="$(CGO_CFLAGS)" go test -v github.com/status-im/status-go/rpc -tags gowaku_skip_migrations,nimbus_light_client -run ^TestProxySuite$$ -testify.m TestRun -ldflags $(LDFLAGS)
 
 
-run-integration-tests: SHELL := /bin/sh
```

Comment:
@jakubgs this is because of `codecov` CLI tool, which we have as nix derivation:
https://github.com/status-im/status-go/blob/ea290d97ef1cc18fac34321a6a12e2ae214612fc/nix/pkgs/codecov-cli/default.nix#L16

It's the only thing, but I didn't see a simple way to avoid it here 🤷